spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -489,13 +486,9 @@ public SparkContext createSparkContext_1() {
     if (execUri != null) {
       conf.set("spark.executor.uri", execUri);
     }
-    if (System.getenv("SPARK_HOME") != null) {
-      conf.setSparkHome(System.getenv("SPARK_HOME"));
```

Comment:
actually, spark-submit should be able to set SPARK_HOME itself too
https://github.com/apache/spark/blob/master/bin/spark-submit#L20

but I think Zeppelin requires SPARK_HOME to be set before calling spark-submit https://github.com/apache/zeppelin/blob/a3c78a4f49653ee66b0316c694bf57bf1c38a53b/bin/interpreter.sh#L108